app/assets/stylesheets/shared_styles/elements/buttons.scss

```diff
@@ -137,13 +137,17 @@
     }
   }
 
+  &.disabled,
   &:disabled {
-    border: $border-tertiary;
     color: $color-silver-chalice;
+    cursor: not-allowed;
     opacity: .5;
+    pointer-events: auto !important;
```

Comment:
Why it here?

Reply:
to overwrite bootstrap

Reply:
You will break boostrap

Reply:
It is important for 'disabled' links, don't do it

Reply:
:+1: